frontend/src/app/components/ExperimentTourDialog/index.js

```diff
@@ -6,44 +6,83 @@ import React from "react";
 
 import StepModal from "../StepModal";
 
+import type { InstalledExperiments } from "../../reducers/addon";
+
 import { experimentL10nId } from "../../lib/utils";
 
 type ExperimentTourDialogProps = {
+  hasAddon?: boolean,
+  installed?: InstalledExperiments,
   experiment: Object,
   isExperimentEnabled: Function,
   onCancel: Function,
   onComplete: Function,
-  sendToGA: Function
+  sendToGA: Function,
+  isFeatured?: boolean
 }
 
 export default class ExperimentTourDialog extends React.Component {
   props: ExperimentTourDialogProps
 
+  applyAdditionalMetricsArgs(gaArgs: Object) {
+    const { installed, hasAddon } = this.props;
+    return Object.assign(gaArgs, {
+      dimension1: hasAddon,
+      dimension2: installed ? Object.keys(installed).length > 0 : false,
+      dimension3: installed ? Object.keys(installed).length : 0,
+      dimension13: "Featured Experiment"
+    });
+  }
+
   stepNextPing = (newStep: number) => {
-    this.props.sendToGA("event", {
+    const { sendToGA, isFeatured, experiment } = this.props;
+    let gaArgs = {
       eventCategory: "ExperimentDetailsPage Interactions",
       eventAction: "button click",
       eventLabel: `forward to step ${newStep}`,
-      dimension11: this.props.experiment.slug
-    });
+      dimension11: experiment.slug,
+      dimension13: "Experiment Detail"
+    };
+
+    if (isFeatured) {
+      gaArgs = this.applyAdditionalMetricsArgs;
```

Comment:
I'm a bit confused here... It looks like `this.applyAdditionalMetricsArgs` is actually a method (defined above on L27), and takes a `gaArgs` object argument. Is this just calling `applyAdditionalMetricsArgs()` with a null/undefined `gaArgs` argument? Or should this be something like `gaArgs = this.applyAdditionalMetricsArgs(gaArgs)` which will inject additional properties into the `gaArgs` defined above on L39?

Reply:
Yeah, this is exactly right.

Reply:
@pdehaan thanks for catching this, I accidently pulled off the actual calling of the function when I switched away from using `.bind(this)`